services/main-frontend/src/pages/account-deleted.tsx

```diff
@@ -0,0 +1,44 @@
+import { css } from "@emotion/css"
+import { Envelope } from "@vectopus/atlas-icons-react"
+import Link from "next/link"
+import { useTranslation } from "react-i18next"
+
+import Button from "@/shared-module/common/components/Button"
+import { baseTheme } from "@/shared-module/common/styles"
+
+const AccountDeletedPage: React.FC<React.PropsWithChildren<unknown>> = () => {
```

Comment:
No need for` PropsWithChildren` since this one does not use the `children` prop. Also no need to use `unknown` for the props since we know which props this component takes (nothing).

So this one could be simplified just to

```jsx
const AccountDeletedPage: React.FC = () => {
```